src/dataset.rs

```diff
@@ -327,6 +354,10 @@ impl Dataset {
         }
         Ok(transformation)
     }
+
+    pub fn has_capability(&self, capability: DatasetCapability) -> bool {
+        unsafe { gdal_sys::GDALDatasetTestCapability(self.c_dataset(), capability.0.as_ptr()) == 1 }
```

Comment:
Well, I do hope that `TRUE` is 1 and not `0xFFFF_FFFF` :sweat_smile:.

Reply:
That's what test cases are for. It's working for now.